src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,97 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
dont use Arrays for learning purpose: 

see common mistakes: https://github.com/mate-academy/jv-array-list/blob/master/checklist.md